chat/Cargo.toml

```diff
@@ -25,6 +26,10 @@ tracing = "0.1.40"
 tracing-appender = "0.2.3"
 tracing-subscriber = { version = "0.3.18", features = ["env-filter"] }
 unicode-segmentation = "1.12.0"
+rand = "0.8.5"
+rusqlite = { version = "0.32.1", features = ["bundled"] }
+blake3 = "1.5.4"
+plonky2 = { git = "https://github.com/mir-protocol/plonky2" }
```

Comment:
We should move this to the workspace too so we can pin it to this specific fork

Reply:
eh I'll just pin it to the commit hash

Reply:
oh, I see what ur saying

like we move the dep to the workspace

Reply:
yes agreed thats better